internal/generate/sweeperregistration/file.tmpl

```diff
@@ -0,0 +1,17 @@
+// Code generated by internal/generate/sweeperregistration/main.go; DO NOT EDIT.
+
+package {{ .PackageName }}
+
+import (
+	"context"
+
+{{ range .Services }}
+	"github.com/hashicorp/terraform-provider-aws/internal/service/{{ .ProviderPackage }}"
+{{- end }}
+)
+
+func registerSweepers(context.Context) {
+{{- range .Services }}
+	{{ .ProviderPackage }}.RegisterSweepers()
```

Comment:
Do we want to propagate `context.Context` here?

Reply:
It's not needed at this point